document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/document/description/impl/RetrieveApiEnumerationDescriptionImpl.java

```diff
@@ -31,8 +31,10 @@ public String getApiEnumerationDescription(String fileName, String identifier, S
 
         String description = "";
 
+        LOG.info("obtaining file for api enumerations with file name: " + descriptionsFile);
         try (InputStream inputStream = new FileInputStream(descriptionsFile)) {
 
+            LOG.info("The file: " + descriptionsFile + " has been found, obtaining descriptions");
```

Comment:
Can you use infoContext for the info logs please? It'll be easier to link that to the errors.